postgres/cloudpostgres/cloudpostgres.go

```diff
@@ -55,6 +58,45 @@ type Params struct {
 	TraceOpts []ocsql.TraceOption
 }
 
+// URLOpener opens GCP PostgreSQL URLs
+// like "cloudpostgres://user:password@myproject/us-central1/instanceId/mydb".
+type URLOpener struct {
+	// TraceOpts contains options for OpenCensus.
+	TraceOpts  []ocsql.TraceOption
+	CertSource *certs.RemoteCertSource
+}
+
+// Scheme is the URL scheme cloudpostgres registers its URLOpener under on
+// postgres.DefaultMux.
```

Comment:
To match other packages' docs, please add info about this to the package documentation. See [gcsblob](https://godoc.org/gocloud.dev/blob/gcsblob#hdr-URLs) for an example.